CONTRIBUTING.md

```diff
@@ -9,94 +10,145 @@ it can be incorporated into the `main` branch.
+pip uninstall mne
+pip install -e .
+```
 
-### Install Python packages required to run tests
+This will uninstall the current MNE-Python installation and instead install the MNE-Python
+development version, including access to several internal test files that are needed
+for `mne-bids` as well.
 
-You can install the testing requirements using the `test_requirements.txt` file
-from the root of the mne-bids repository.
+### Install the development version of MNE-BIDS
 
-    $ pip install -r test_requirements.txt
+Now [fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) the `mne-bids` repository.
+Then, `git clone` your fork and install it in "editable" mode.
 
-### Install the BIDS validator
+```Shell
+git clone https://github.com/<your-GitHub-username>/mne-bids
+pip install -e ./mne-bids
+```
 
-Finally, it is necessary to install the
-[BIDS validator](https://github.com/bids-standard/bids-validator). The outputs
-of MNE-BIDS are run through the BIDS validator to check if the conversion
-worked properly and produced datasets that conforms to the BIDS specifications.
+You should now have both the `mne` and `mne-bids` development versions available in your Python environment.
 
-You will need the `command line version` of the validator.
+### Install additional Python packages required for development
 
-#### Global (system-wide) installation
+Navigate to the root of the `mne-bids` repository and call:
 
-- First, install [Node.js](https://nodejs.org/en/).
-- For installing the **stable** version of `bids-validator`, please follow the
-instructions as detailed in the README of the bids-validator repository.
-- For installing the **development** version of `bids-validator`, see [here](https://github.com/bids-standard/bids-validator/blob/master/CONTRIBUTING.md#using-the-development-version-of-bids-validator).
+```Shell
+pip install -r test_requirements.txt
+pip install -r doc/requirements.txt
+```
 
-Test your installation by running:
+This will install several packages to run tests, and build the documentation for `mne-bids`.
 
-    $ bids-validator --version
+### Install the BIDS validator
 
-#### Local (per-user) development installation
+For a complete development setup of `mne-bids`, it is necessary to install the
+[BIDS validator](https://github.com/bids-standard/bids-validator).
+The outputs of `mne-bids` are run through the BIDS validator to check if the conversion
+worked properly and produced datasets conform to the
+[BIDS specification](https://bids-specification.readthedocs.io/en/stable/).
 
-Install [Node.js](https://nodejs.org/en/). If you're use `conda`, you can
-install the `nodejs` package from `conda-forge` by running
-`conda install -c conda-forge nodejs`.
+To install the **stable** version of `bids-validator`, please follow the instructions
+in the [bids-validator README](https://github.com/bids-standard/bids-validator#quickstart).
+You will need the "command line version" of the bids-validator.
```

Comment:
Can we not simply tell them how to do it?

```shell
conda install -c conda-forge nodejs
npm i -g bids-validator
```

Reply:
Yes, I think for most beginners this would be fine. I added it